roles/kubernetes/node/templates/kubelet.standard.env.j2

```diff
@@ -87,11 +87,12 @@ KUBELET_HOSTNAME="--hostname-override={{ kube_override_hostname }}"
 {%   if not standalone_kubelet|bool %}
 {%     do role_node_labels.append('node-role.kubernetes.io/node=true') %}
 {%   endif %}
-{% elif inventory_hostname in groups['kube-ingress']|default([]) %}
-{%   set node_labels %}--node-labels=node-role.kubernetes.io/ingress=true{% endset %}
 {% else %}
 {%   do role_node_labels.append('node-role.kubernetes.io/node=true') %}
 {% endif %}
+{% if inventory_hostname in groups['kube-ingress']|default([]) %}
```

Comment:
@woopstar ok i get your point, it should just become a MUST for ingress-nginx only, but NOT A MUST for overall grouping. That's why previous build failed.